sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/BlobChangeFeedClient.cs

```diff
@@ -111,6 +111,28 @@ public BlobChangeFeedClient(Uri serviceUri, StorageSharedKeyCredential credentia
             _blobServiceClient = new BlobServiceClient(serviceUri, credential, options);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BlobChangeFeedClient"/>
+        /// class.
```

Comment:
We should add a blurb to all these doc comments telling people to only use this overload if they need to roll SAS signatures.

Reply:
added remarks.